### PR TITLE
updated VMware list and removed some duplicates

### PR DESCRIPTION
**Pull request by Goldshop**
Thank you for your contribution! Some pointers to get it merged quickly:

For contributions in `software/`:

  - [x] PR Title: Please use "Add <vendor/product name>" (instead of "Update README.md")
  - [x] Status: please select a value from the status table at the top
  - [x] Version: Status Vulnerable / Workaround? -> List vulnerable versions.
             Status Fix?                     -> List fixed versions.
  - [x] Links: make sure you link to a public statement by the developer.
    Alternatively, include and link a file in the `software/vendor-statements/`
    directory
  - [x] Please mind the sorting

### CODE REVIEW

File: software/software_list_v.md [hunk 1]
```diff
@@ -72,74 +72,76 @@
...
+|VMware|Horizon Agents Installer|patch for 20.3.0, 20.3.1| Not vuln | Fix | Fix | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
+|VMware|Horizon Cloud Connector|1.x| Not vuln | Vulnerable | Vulnerable | Not vuln | Upgrade to 2.1.2 required |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
+|VMware|Horizon Cloud Connector|2.1.2| Not vuln | Fix | Fix | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html) [fix](https://customerconnect.vmware.com/downloads/details?downloadGroup=HCS-CC-210&amp;productId=716&amp;rPId=79131#product_downloads)|
+|VMware|Horizon DaaS|9.1.x,9.0.x| Not vuln | Workaround | Workaround | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html) [workaround](https://kb.vmware.com/s/article/87101)|
+|VMware|Identity Manager|patches for 3.3.5, 3.3.4, 3.3.3| Not vuln | Fix | Fix | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html) [KB] (https://kb.vmware.com/s/article/87185)|
```
Comment (tschmidtb51) [inline software/software_list_v.md:94]:
```suggestion
|VMware|Identity Manager|3.3.6| Not vuln | Fix | Fix | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html) [KB] (https://kb.vmware.com/s/article/87185)|

File: software/software_list_v.md [hunk 2]
```diff
@@ -72,74 +72,76 @@
...
+|VMware|Integrated OpenStack|7.2| Not vuln | Fix | Fix | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html) [Fix](https://docs.vmware.com/en/VMware-Integrated-OpenStack/7.2/rn/VMware-Integrated-OpenStack-72-Release-Notes.html)|
+|VMware|NSX Data Center for vSphere|6.4.12| Not vuln |Fix|Fix| Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
+|VMware|NSX-T Data Center|3.1.3.5, 3.0.3.1, 2.5.3.4| Not vuln |Fix|Fix| Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
+|VMware|NSX-T Intelligence Appliance|1.2.x, 1.1.x| Not vuln |Fix|Fix| Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
```
Comment (tschmidtb51) [inline software/software_list_v.md:98]: I didn't find that one. Did you mean?
```suggestion
|VMware|NSX Intelligence |1.2.1.1| Not vuln |Fix|Fix| Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
```
If so, please adopt sorting.

File: software/software_list_v.md [hunk 3]
```diff
@@ -72,74 +72,76 @@
...
+|VMware|SD-WAN VCO|4.x| Not vuln | Fix | Fix | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html) [fix](https://kb.vmware.com/s/article/87158)|
```
Comment (tschmidtb51) [inline software/software_list_v.md:99]: |VMware|SD-WAN VCO|4.x| Not vuln | Fix | Fix | Not vuln | Check [KB 87158](https://kb.vmware.com/s/article/87158) for applicable builds|[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html) [fix](https://kb.vmware.com/s/article/87158)|

File: software/software_list_v.md [hunk 4]
```diff
@@ -72,74 +72,76 @@
...
+|VMware|Single Sign-On for VMware Tanzu Application Service|1.14.6| Not vuln | Fix | Fix | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html) [fix](https://network.tanzu.vmware.com/products/pivotal_single_sign-on_service#/releases/1012467)|
+|VMware|Site Recovery Manager, vSphere Replication|8.5.0.2, 8.4.0.4, 8.3.1.5| Not vuln | Fix | Fix | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
 |VMware|Skyline Collector virtual appliance| |Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://kb.vmware.com/s/article/87068)|
-|VMware|Smart Assurance M&R|6.8u5, 7.0u8, 7.2.0.1| | Vulnerable | Vulnerable | | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
+|VMware|Smart Assurance M&R|6.8u5, 7.0u8, 7.2.0.1| Not vuln | Vulnerable | Vulnerable | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
```
Comment (tschmidtb51) [inline software/software_list_v.md:103]:
```suggestion
|VMware|Smart Assurance M&R|6.8u5, 7.0u8, 7.2.0.1| Not vuln | Workaround | Workaround | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|

File: software/software_list_v.md [hunk 5]
```diff
@@ -72,74 +72,76 @@
...
 |VMware|Smart Assurance NCM|10.1.6| | Vulnerable | Vulnerable | | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
```
Comment (tschmidtb51) [inline software/software_list_v.md:104]:
```suggestion
|VMware|Smart Assurance NCM|10.1.6| Not vuln | Workaround | Workaround | | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|

File: software/software_list_v.md [hunk 6]
```diff
@@ -72,74 +72,76 @@
...
-|VMware|Unified Access Gateway (UAG)|3.x| |Workaround|Workaround| |Upgrade to 2111.1 required |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
-|VMware|vCenter Cloud Gateway|1.x|Not vuln|Workaround| | | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html) [workaround](https://kb.vmware.com/s/article/87081)|
-|VMware|vCenter Server - OVA|7.x, 6.7.x, 6.5.x| |Vulnerable| | |<a href="https://kb.vmware.com/s/article/87081" rel="nofollow">Workaround @ KB87081 (vmware.com)</a>|[VMSA-2021-0028.4 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
-|VMware|vCenter Server - Windows|6.7.x, 6.5.x| |Vulnerable| | |<a href="https://kb.vmware.com/s/article/87096" rel="nofollow">Workaround @ KB87096 (vmware.com)</a>|[VMSA-2021-0028.4 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
+|VMware|Smart Assurance SAM [Service Assurance Manager]|10.1.0.x, 10.1.2, 10.1.5| Not vuln | Vulnerable | Vulnerable | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
```
Comment (tschmidtb51) [inline software/software_list_v.md:105]:
```suggestion
|VMware|Smart Assurance SAM [Service Assurance Manager]|10.1.5.5 | Not vuln | Vulnerable | Vulnerable | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|

File: software/software_list_v.md [hunk 7]
```diff
@@ -72,74 +72,76 @@
...
+|VMware|Tanzu Kubernetes Grid Integrated Edition |1.13.1, 1.10.8| Not vuln | Fix | Fix | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
+|VMware|Tanzu Observability by Wavefront Nozzle |3.0.4| Not vuln | Fix | Fix | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html) [fix](https://network.pivotal.io/products/wavefront-nozzle)|
+|VMware|Tanzu Observability Proxy|10.12| Not vuln | Fix | Fix | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
+|VMware|Tanzu Operations Manager|2.8.18, 2.9.25, 2.10.24| Not vuln | Fix | Fix | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)  [workaround](https://community.pivotal.io/s/article/5004y00001mPn2N1639255611105?language=en_US) [fix](https://network.pivotal.io/products/ops-manager/)|
```
Comment (tschmidtb51) [inline software/software_list_v.md:116]: According to the second link:
```suggestion
|VMware|Tanzu Operations Manager|2.8.19, 2.9.26, 2.10.25| Not vuln | Fix | Fix | Fix | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html) [workaround](https://community.pivotal.io/s/article/5004y00001mPn2N1639255611105?language=en_US) [fix](https://network.pivotal.io/products/ops-manager/)|

File: software/software_list_v.md [hunk 8]
```diff
@@ -72,74 +72,76 @@
...
+|VMware|Tanzu Scheduler|1.6.1| Not vuln | Fix | Fix | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
+|VMware|Tanzu SQL with MySQL for VMs|2.x, 1.x| Not vuln |Not vuln|Not vuln | Not vuln | |[source](https://kb.vmware.com/s/article/87068)|
```
Comment (tschmidtb51) [inline software/software_list_v.md:118]: Does that only apply for version 2.x and 1.x or for all versions?

File: software/software_list_v.md [hunk 9]
```diff
@@ -72,74 +72,76 @@
...
+|VMware|Telco Cloud Automation|2.x, 1.x| Not vuln |Not vuln| Not vuln | Not vuln | |[source](https://kb.vmware.com/s/article/87068)|
```
Comment (tschmidtb51) [inline software/software_list_v.md:119]: Same question here.

File: software/software_list_v.md [hunk 10]
```diff
@@ -72,74 +72,76 @@
...
+|VMware|Unified Access Gateway (UAG)|20.x| Not vuln |Workaround|Workaround| Not vuln |Upgrade to 2111.1 required |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
+|VMware|Unified Access Gateway (UAG)|2111.1| Not vuln |Fix|Fix| Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
+|VMware|Unified Access Gateway (UAG)|3.x| Not vuln |Workaround|Workaround| Not vuln |Upgrade to 2111.1 required |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
+|VMware|vCenter Cloud Gateway|1.x|Not vuln|Workaround|Workaround| Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html) [workaround](https://kb.vmware.com/s/article/87081)|
+|VMware|vCenter Server - OVA|6.x| Not vuln | Workaround | Workaround | Not vuln |<a href="https://kb.vmware.com/s/article/87081" rel="nofollow">Workaround @ KB87081 (vmware.com)</a>|[VMSA-2021-0028.4 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
```
Comment (tschmidtb51) [inline software/software_list_v.md:125]:
```suggestion
|VMware|vCenter Server - OVA|6.x| Not vuln | Workaround | Workaround | Not vuln |[VMSA-2021-0028.4 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html) [Workaround](https://kb.vmware.com/s/article/87081)|

File: software/software_list_v.md [hunk 11]
```diff
@@ -72,74 +72,76 @@
...
+|VMware|vCenter Server - OVA|7.0U3c| Not vuln | Fix | Fix | Not vuln |<a href="https://kb.vmware.com/s/article/87081" rel="nofollow">Workaround @ KB87081 (vmware.com)</a>|[VMSA-2021-0028.4 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html) [fix](https://docs.vmware.com/en/VMware-vSphere/7.0/rn/vsphere-vcenter-server-70u3c-release-notes.html)|
```
Comment (tschmidtb51) [inline software/software_list_v.md:126]:
```suggestion
|VMware|vCenter Server - OVA|7.0U3c| Not vuln | Fix | Fix | Not vuln | |[VMSA-2021-0028.4 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html) [Workaround](https://kb.vmware.com/s/article/87081) [fix](https://docs.vmware.com/en/VMware-vSphere/7.0/rn/vsphere-vcenter-server-70u3c-release-notes.html)|

File: software/software_list_v.md [hunk 12]
```diff
@@ -72,74 +72,76 @@
...
+|VMware|vCenter Server - Windows|6.7.x, 6.5.x| Not vuln | Workaround | Workaround | Not vuln |<a href="https://kb.vmware.com/s/article/87096" rel="nofollow">Workaround @ KB87096 (vmware.com)</a>|[VMSA-2021-0028.4 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
```
Comment (tschmidtb51) [inline software/software_list_v.md:127]:
```suggestion
|VMware|vCenter Server - Windows|6.7.x, 6.5.x| Not vuln | Workaround | Workaround | Not vuln | |[VMSA-2021-0028.4 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html) [Workaround](https://kb.vmware.com/s/article/87096)|

File: software/software_list_v.md [hunk 13]
```diff
@@ -72,74 +72,76 @@
...
+|VMware|vRealize Business for Cloud|7.x| Not vuln | Vulnerable | Vulnerable | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
+|VMware|vRealize Lifecycle Manager|8.6.2| Not vuln | Fix | Fix | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html) [workaround](https://kb.vmware.com/s/article/87097) [fix](https://docs.vmware.com/en/VMware-vRealize-Suite-Lifecycle-Manager/8.6/rn/VMware-vRealize-Suite-Lifecycle-Manager-862-Release-Notes.html)|
+|VMware|vRealize Log Insight|8.2, 8.3, 8.4| Not vuln | Vulnerable | Vulnerable | Not vuln | Upgrade to 8.6.2 required |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
+|VMware|vRealize Log Insight|8.6.2| Not vuln | Fix | Fix | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
+|VMware|vRealize Network Insight|5.3, 6.x| Not vuln | Vulnerable | Vulnerable | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|
```
Comment (tschmidtb51) [inline software/software_list_v.md:137]:
```suggestion
|VMware|vRealize Network Insight|6.5| Not vuln | Fix | Fix | Not vuln | |[VMSA-2021-0028.1 (vmware.com)](https://www.vmware.com/security/advisories/VMSA-2021-0028.html)|

**Review by tschmidtb51 — CHANGES_REQUESTED** [review]
@Goldshop Thank you for your contribution.

Could you please a have a look at my comments? Some changes are necessary before we can merge.

**Comment by Goldshop** [comment]
All changes have been completed

**Review by tschmidtb51 — APPROVED** [review]
Thank you for your contribution.

**Pull request merged**